core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -495,4 +514,76 @@ class ConsoleConsumerTest {
       Exit.resetExitProcedure()
     }
   }
+
+  /**
+    * There is no test for the default key and value deserializer properties
+    */
+  @Test(expected = classOf[ShouldExitException])
+  def shouldExitOnUnrecognisedProperty() {
+    val expectedMessage = Some("Unrecognised arguments ({not=ok, bad.property=not.good}) passed to kafka.tools.DefaultMessageFormatter.")
+    Exit.setExitProcedure((_, message) => {
+      assertEquals("Console consumer should exit with help information with bad properties listed", expectedMessage, message)
+      throw new ShouldExitException() // Code should exit with help information
+    }
+    )
```

Comment:
nit: formatting, the parentheses could be placed on top to avoid an extra new line